Announcement: moderation team (#18)
codewars/blog: content/2021/2021-01-22_moderators/index.md

+++
title = "Codewars moderation team"
description = "Codewars staff reinforced by a team of community members!"

[taxonomies]
tags = ["announcement", "moderation", "community"]
+++

> Until recently, Codewars maintainers and admins were overwhelmed by amount of tasks related not only to the maintenance of the platform, fixing bugs, and adding new features, but also by issues related to Codewars content and users (poor **@Kazk** \*hug\*). Sometimes it was just too much for them.

Reviewer: ```suggestion
Until recently, Codewars maintainers and admins were overwhelmed by the number of tasks related not only to the maintenance of the platform, fixing bugs, and adding new features but also by issues related to Codewars content and users (poor **@Kazk** \*hug\*). Sometimes it was just too much for them.
```

> Until recently, Codewars maintainers and admins were overwhelmed by the number of tasks related not only to the maintenance of the platform, fixing bugs, and adding new features but also by issues related to Codewars content and users (poor **@Kazk** \*hug\*). Sometimes it was just too much for them.

Reviewer 2: Thanks for doing this.
But like the first post, it sounds really awkward if I (maintainer/admin) post this because it's using third person to talk about myself. Especially "poor me" part.

I'll consider prioritizing adding the author field and letting you guys post under a separate name.

Author: My idea was to make the Codewars platform talking, that's why "we", as in "we, the Codewars". Maybe it's just not how it should be done?

If you want to change it to first person voice and post as "me, Kazk, the maintainer", then go ahead, and I can sketch other posts in similar way too.

> I'll consider prioritizing adding the author field and letting you guys post under a separate name.

Well, I do not know about others, but I will definitely not publish anything into public without a proper authorization and language review. I can help with writing as much as I am able to, but I do not feel confident enough to do the whole publication process :)

Reviewer 2: Oh, I always thought `we == The Codewars Team`. Never thought that the platform was talking. Personification? I don't think we do that for blogs, but maybe I'm misunderstanding you.

> If you want to change it to first person voice and post as "me, Kazk, the maintainer", then go ahead, and I can sketch other posts in similar way too.

No, I wouldn't want to focus on myself like that.

> Well, I do not know about others, but I will definitely not publish anything into public without a proper authorization and language review. I can help with writing as much as I am able to, but I do not feel confident enough to do the whole publication process :)

Nothing changes to what you're doing. I just thought it'll be easier for you (and others) to write from your perspective and publish as the moderator team. If it's not, then we'll just continue posting without author.

> To take some of the burden off of their backs, a group of trusted community members has been selected and granted a title of **moderator**, forming the community support team: a team which would help admins handling some tasks related to users and content. Thanks to recent refactorings it was finally possible to introduce additional tools and features which can be used by priviledged members to perform additional tasks.

Reviewer: ```suggestion
To take some of the burdens off of their backs, a group of trusted community members has been selected and granted a title of **moderator**, forming the community support team: a team that would help admins handling some tasks related to users and content. Thanks to recent refactorings, it was finally possible to introduce additional tools and features that can be used by privileged members to perform additional tasks.
```

> Moderators are meant to be the first line of support for Codewars users and intermediaries between community, and administrators. They have access to tools which help them with handling bad quality content, problematic and dishonest users, they can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature will be added which would be in some way helpful for regular users, we will announce it with a dedicated post.

Reviewer: ```suggestion
Moderators are meant to be the first line of support for Codewars users and intermediaries between the community and administrators. They have access to tools that help them with handling bad-quality content and problematic and dishonest users, they can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature will be added which would be in some way helpful for regular users, we will announce it with a dedicated post.
```

What about mentioning about for those who joined the community-support team, they can also manage issues on the codewars.com repo?

Author: I am not sure if its worth mentioning specifically here in the announcement, but i will add a note to "moderator features" reference page.

> Effectivity of moderators' work heavily relies on their communication with community members. After all, they cannot help if they do not know that something is wrong, right? Whenever you encounter some issue like a kata with some serious flaw, a solution which should not be there, a user causing some grief, just [let them know](https://docs.codewars.com/community/moderation/#how-to-get-in-touch). [This reference page](https://docs.codewars.com/references/moderator-features/) lists the ways moderators can possibly help you. A dedicated [help page](https://docs.codewars.com/community/moderation/#moderators) describes details on current state of the moderation team, list of users, and their responsibilities. 

Reviewer: ```suggestion
Effectivity of the moderators' work heavily relies on their communication with community members. After all, they cannot help if they do not know that something is wrong, right? Whenever you encounter some issue like a kata with some serious flaw, a solution that should not be there, a user causing some grief, just [let them know](https://docs.codewars.com/community/moderation/#how-to-get-in-touch). [This reference page](https://docs.codewars.com/references/moderator-features/) lists the ways moderators can help you. A dedicated [help page](https://docs.codewars.com/community/moderation/#moderators) describes details on the current state of the moderation team, list of users, and their responsibilities. 
```

> Effectivity of the moderators' work heavily relies on their communication with community members. After all, they cannot help if they do not know that something is wrong, right? Whenever you encounter some issue like a kata with some serious flaw, a solution that should not be there, a user causing some grief, just [let them know](https://docs.codewars.com/community/moderation/#how-to-get-in-touch). [This reference page](https://docs.codewars.com/references/moderator-features/) lists the ways moderators can help you. A dedicated [help page](https://docs.codewars.com/community/moderation/#moderators) describes details on the current state of the moderation team, list of users, and their responsibilities. 

Reviewer 3: ```suggestion
The effectiveness of moderators' work heavily relies on their communication with community members. After all, they cannot help if they do not know that something is wrong, right? Whenever you encounter some issue like a kata with some serious flaw, a solution that should not be there, a user causing some grief, just [let them know](https://docs.codewars.com/community/moderation/#how-to-get-in-touch). [This reference page](https://docs.codewars.com/references/moderator-features/) lists the ways moderators can help you. A dedicated [help page](https://docs.codewars.com/community/moderation/#moderators) describes details on the current state of the moderation team, list of users, and their responsibilities.
```

> We hope that support from the moderators will improve handling of problems which bugged Codewars for so long. They will help with communication between community members and staff, bringing them closer together. Moderators will also relieve administrators from some of the most mundane tasks, letting them focus on providing new features, fixing bugs, and improving Codewars overall.

Reviewer: ```suggestion
We hope that support from the moderators will improve the handling of problems that bugged Codewars for so long. They will help with communication between community members and staff, bringing them closer together. Moderators will also relieve administrators from some of the most mundane tasks, letting them focus on providing new features, fixing bugs, and improving Codewars overall.
```

> To take some of the burdens off of their backs, a group of trusted community members has been selected and granted a title of **moderator**, forming the community support team: a team that would help admins handling some tasks related to users and content. Thanks to recent refactorings, it was finally possible to introduce additional tools and features that can be used by privileged members to perform additional tasks.

Reviewer 3: ```suggestion
To take some of the burden off their backs, a group of trusted community members has been selected and granted a title of **moderator**, forming the community support team: a team that would help admins handle certain tasks related to users and content. Thanks to recent efforts in refactoring, it was finally possible to introduce additional tools and features that can be used by privileged members to perform additional tasks.
```

> Moderators are meant to be the first line of support for Codewars users and intermediaries between the community and administrators. They have access to tools that help them with handling bad-quality content and problematic and dishonest users, they can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature will be added which would be in some way helpful for regular users, we will announce it with a dedicated post.

Reviewer 3: ```suggestion
Moderators are meant to be the first line of support for Codewars users and intermediaries between the community and administrators. They have access to tools that help them with handling bad-quality content, as well as problematic and dishonest users. They can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature is added which would be helpful for regular users in some way, we will announce it with a dedicated post.
```
